File: demo/java-demo/src/test/java/com/alibaba/demo/one2multi/BSvcMock.java
package com.alibaba.demo.one2multi;

import com.alibaba.testable.core.annotation.MockInvoke;

public class BSvcMock {

    @MockInvoke(targetClass = String.class, targetMethod = "format")
    public String b_format(String format, Object... args) {
        return "b_mock";
    }

}

File: demo/kotlin-demo/src/test/kotlin/com/alibaba/demo/one2multi/BSvcMock.kt
package com.alibaba.demo.one2multi

import com.alibaba.testable.core.annotation.MockMethod

class BSvcMock {

    @MockMethod(targetClass = String::class, targetMethod = "format")
    fun b_format(format: String, vararg args: Any?): String {
        return "b_mock"
    }

}
